### DISCUSSION TITLE
Does pixijs canvas support 1000 * 100000

### DISCUSSION THREAD

**Discussion by a6965921** [pixijs/pixijs#7412] [Q&A]
Does pixijs canvas support 1000 * 100000

**Comment by bigtimebuddy** [comment]
No. Browsers limit the max canvas size. 

### Chrome
Maximum height/width: 32,767 pixels
Maximum area: 268,435,456 pixels (e.g., 16,384 x 16,384)

### Firefox
Maximum height/width: 32,767 pixels
Maximum area: 472,907,776 pixels (e.g., 22,528 x 20,992)
